.vscode/extensions.json

```diff
@@ -0,0 +1,26 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+    // Java Extension Pack
```

Comment:
Indentation on this is kinda nuts and inconsistent with out editor config settings.

Also, the highlighting/coloring is all weird because JSON doesn't technically support comments.

Reply:
Some additional extensions to consider adding:

- I talk about Gradle Tasks in the write-up so I would like to include it. For now, at least, it will show them some useful things about the large number of tasks Gradle supports.
- EditorConfig For VS Code since we're using EditorConfig throughout the semester.
-  Markdown-all-in-one is a very nice extension that provides a number of very nice Markdown tools.
- GitHub Markdown Preview is a pack that allows the VS Code Markdown rendering to match that on GitHub, which is nice.

Reply:
I added these (except EditorConfig, which was already there) and another Gradle extension @floogulinc recommended in 34b9d73c90e866f2437fcf5172f5f04957b20003 so I think we're good.